rust/feature-flags/src/server.rs

```diff
@@ -38,8 +41,23 @@ where
         }
     };
 
+    let health = HealthRegistry::new("liveness");
+
+    // TODO - we don't have a more complex health check yet, but we should add e.g. some around DB operations
+    let simple_loop = health
+        .register("simple_loop".to_string(), Duration::from_secs(30))
+        .await;
+    tokio::spawn(liveness_loop(simple_loop));
```

Comment:
we have a TODO for adding a more complex one, but this is a great start for thinking about how to do it.